scripts/migrate-unknown.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+#set -x
+
+TOKEN=none
+ENV=dev
+DRY_RUN=1
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -t)
+      TOKEN=$2
+      shift # past argument
+      shift # past value
+      ;;
+    -e)
+      ENV=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --no-dry-run)
+      DRY_RUN=0
```

Comment:
might be better to just set `REQ_PATH` here rather than having separate variable